index.js

```diff
@@ -0,0 +1,60 @@
+document.addEventListener('DOMContentLoaded', () => {
+// console.log("dom")
+setupDroplist();
+fetchMovieList()
+})
+// add list to drop list
+const setupDroplist = ()=>{
+  const getMovieList = document.querySelector('#droplist')
+  getMovieList.addEventListener('click', fetchMovieList)
+  console.log( "click")
+}
+
+const fetchMovieList =()=>{
+  console.log("fetch was called")
+  fetchAllTitles();
+  // selectedTitle();
+  // fetchYear();
+  // fetchDescription();
+}
+
+const fetchAllTitles=(film)=>{
+  
+  const films =` https://ghibliapi.herokuapp.com/films`
+  // console.log(url)
+  fetch(films)
+  .then(response => response.json())
+  .then(films => {
+    // console.log(films)
+    for (let i = 0; i< films.length; i++) {
+     let titleFilms = films[i].title
+    let options = document.createElement('option');
+    options.value = titleFilms;
+    options.innerText = titleFilms;
+    let select =document.querySelector('#droplist')
+    select.appendChild(options)
+   
+    }
+  })
+  const selectedTitle=(select)=>{
+  
+    let option;
+      for (let i = 0; i< titleFilms.length; i++) {
+       option = select[i]
+      if(option.selected === true){
+        return option
+        console.log(option)
+      }
+      }
+      return option
```

Comment:
I imagine this was going to be your listener for the `change` event (whenever the user selected an option from the list) on the select box. And I see you looked into `option.selected` which is still a valid approach and could definitely help.